### PR TITLE
EmploymentHistory component

### LINKED ISSUE #2934: EmploymentHistoryPage component
https://app.moqups.com/greg@adhocteam.us/l32hJ36b/view/page/ab059b800

### PR DESCRIPTION
**Pull request by jbalboni**
Closes #2934.

Adds the EmploymentHistory page.

### CODE REVIEW

**Comment by jbalboni** [comment]
@alexose Can you take a look?

File: src/js/edu-benefits/utils/validations.js [hunk 1]
```diff
@@ -44,6 +44,10 @@ function isValidYear(value) {
   return Number(value) >= 1900;
 }
 
+function isValidMonths(value) {
+  return Number(value) >= 0;
```
Comment (alexose) [inline src/js/edu-benefits/utils/validations.js]: `&& < 12`?

**Comment by alexose** [comment]
👍 

Will merge after CI

File: src/js/edu-benefits/utils/validations.js (hunk 1 above)
Comment (jbalboni) [inline src/js/edu-benefits/utils/validations.js]: This might make more sense as `isValidPositiveNumber`. It's for a standalone months field.

**Pull request merged**